schemas/2015-01-01/deploymentTemplate.json

```diff
@@ -118,7 +118,9 @@
                   { "$ref": "http://schema.management.azure.com/schemas/2016-04-01/Microsoft.Network.json#/resourceDefinitions/dnszones_PTR"},
                   { "$ref": "http://schema.management.azure.com/schemas/2016-04-01/Microsoft.Network.json#/resourceDefinitions/dnszones_SOA"},
                   { "$ref": "http://schema.management.azure.com/schemas/2016-04-01/Microsoft.Network.json#/resourceDefinitions/dnszones_SRV"},
-                  { "$ref": "http://schema.management.azure.com/schemas/2016-04-01/Microsoft.Network.json#/resourceDefinitions/dnszones_TXT"}
+                  { "$ref": "http://schema.management.azure.com/schemas/2016-04-01/Microsoft.Network.json#/resourceDefinitions/dnszones_TXT"},
+                  { "$ref": "http://schema.management.azure.com/schemas/2016-03-01/Microsoft.Scheduler.json#/resourceDefinitions/jobCollections"},
+                  { "$ref": "http://schema.management.azure.com/schemas/2016-03-01/Microsoft.Scheduler.json#/resourceDefinitions/jobCollections_jobs"}
```

Comment:
Don't only top level resources get exposed here?  Job is not an ARM registered resource (only job collection).